src/main/java/org/corpus_tools/peppermodules/TEIModules/TEIMapper.java

```diff
@@ -801,15 +801,17 @@ public void characters(char ch[], int start, int length) {
 				for (int i = start; i < start + length; i++) {
 					tempstr.append(ch[i]);
 				}
-				txt.append(tempstr.toString().trim());
+//				txt.append(tempstr.toString().trim());
+				txt.append(tempstr.toString().replaceAll("\\s+", " "));
 			}
 
 			if (metadata) {
 				StringBuilder tempstr = new StringBuilder();
 				for (int i = start; i < start + length; i++) {
 					tempstr.append(ch[i]);
 				}
-				meta_txt.append(tempstr.toString().trim());
+//				meta_txt.append(tempstr.toString().trim());
+				meta_txt.append(tempstr.toString().replaceAll("\\s+", " "));
```

Comment:
Are these two lines (805,814) necessary for the the pull request itself or are they connected to the other pull request #27 ?

I think it is OK to leave these commits in, but I would wait to merge this pull request until the other one is merged, too.